fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -1462,11 +1463,9 @@ Future<RangeResult> GlobalConfigImpl::getRange(ReadYourWritesTransaction* ryw,
                                                KeyRangeRef kr,
                                                GetRangeLimits limitsHint) const {
 	RangeResult result;
-
-	auto& globalConfig = GlobalConfig::globalConfig();
 	KeyRangeRef modified =
 	    KeyRangeRef(kr.begin.removePrefix(getKeyRange().begin), kr.end.removePrefix(getKeyRange().begin));
-	std::map<KeyRef, Reference<ConfigValue>> values = globalConfig.get(modified);
+	std::map<KeyRef, Reference<ConfigValue>> values = globalConfig->get(modified);
```

Comment:
I think the conventional way to access this would be to do something like `ryw->getDatabase()->globalConfig->...`. Done that way, we don't have to maintain this pointer as part of the impl object.